_posts/2017-05-15-helm-grpc.md

```diff
@@ -6,7 +6,7 @@ permalink: blog/helmgrpc
 author: Brian Hardock
 company: DEIS
 company-link: https://deis.com/
-thumbnail: http://gabrtv.github.io/deis-dockercon-2014/img/DeisLogo.png
+thumbnail: https://gabrtv.github.io/deis-dockercon-2014/img/DeisLogo.png
```

Comment:
For blogs where we are referencing resources, it makes sense to roll into this change..